doc/tutorials/basic/size.md

```diff
@@ -0,0 +1,385 @@
+data = pd.DataFrame(
+    [
+        ("Monday", 7),
+        ("Tuesday", 4),
+        ("Wednesday", 9),
+        ("Thursday", 4),
+        ("Friday", 4),
+        ("Saturday", 5),
+        ("Sunday", 4),
+    ],
+    columns=["Day", "Wind Speed (m/s)"],
+)
+
+fig = figure(
+    x_range=data["Day"],
+    title="Wind Speed by Day",
+    y_axis_label="Wind Speed (m/s)",
+    sizing_mode='stretch_both',
+)
+fig.vbar(x=data["Day"], top=data["Wind Speed (m/s)"], width=0.5, color="navy", )
+
+md = pn.pane.Markdown(
+    text,  min_width=200, max_width=500,
+)
+settings = pn.Param(md, parameters=["sizing_mode", "min_width", "max_width"])
+pn.Row(fig, md, settings, height=500,).servable()
+```
+
+:::
+
+#### Change the Defaults
+
+:::{note}
+The default `sizing_mode` of Panel is `"fixed"`. We can change the default `sizing_mode` via `pn.extension(sizing_mode=...)` or `pn.config.sizing_mode=...`.
+:::
+
+Let's redo the initial `sizing_mode` exercise, but this time with a default `sizing_mode` of `"stretch_width"`.
+
+:::{dropdown} Solution
+
+```{pyodide}
+import pandas as pd
+import panel as pn
+from bokeh.plotting import figure
+
+pn.extension(sizing_mode="stretch_width")
+# or
+# pn.extension()
+# pn.config.sizing_mode="stretch_width"
+
+text = """A *wind turbine* is a renewable energy device that converts the kinetic energy from wind into electricity. It typically consists of a tall tower with large blades attached to a rotor. As the wind blows, it causes the rotor to spin, which in turn rotates a generator to produce electricity. Wind turbines are designed to harness the natural power of the wind and are used to generate clean, sustainable energy. They come in various sizes, from small residential turbines to massive commercial installations, and play a crucial role in reducing greenhouse gas emissions and meeting renewable energy goals."""
+data = pd.DataFrame(
+    [
+        ("Monday", 7),
+        ("Tuesday", 4),
+        ("Wednesday", 9),
+        ("Thursday", 4),
+        ("Friday", 4),
+        ("Saturday", 5),
+        ("Sunday", 4),
+    ],
+    columns=["Day", "Wind Speed (m/s)"],
+)
+
+fig = figure(
+    x_range=data["Day"],
+    title="Wind Speed by Day",
+    y_axis_label="Wind Speed (m/s)",
+    sizing_mode='stretch_both',
+)
+fig.vbar(x=data["Day"], top=data["Wind Speed (m/s)"], width=0.5, color="navy", )
+
+md = pn.pane.Markdown(
+    text,  min_width=200, max_width=500,
+)
+pn.Row(fig, md, height=500,).servable()
+```
+
+:::
+
+### Change the Defaults Temporarily
```

Comment:
I have not because its not been documented anywhere. But now that I know its there I will.

Its really useful for  `sizing_mode`, `width`, `styles` and similar parameters you otherwise would have to put into 3-10 individual components your are defining.

Reply:
Yes. Almost in every app. The default `sizing_mode` of fixed is so *old school*.

Reply:
Not so far because it was not documented. So I did not know about it. But now that I do, I will. It makes it so much easier to change the `sizing_mode`, `width`, `styles` etc. of many components. Often you have 3-10 components that need the same default change.

Reply:
I have not because its not been documented anywhere. But now that I have discovered it, I will. For me its quite useful as changing via `pn.extension` changes things for every page in a multipage app.

But its probably an *intermediate* topic or not *core*. I have removed it.